### PR TITLE
KBV-294-redirect-to-callback-url

### PR DESCRIPTION
**Pull request by MattBidewell**
### What changed
Added a redirection after the addresses have been saved.
### Why did it change

We should redirect back to callback URL with the auth code and state.

### Issue tracking

- [KBV-294](https://govukverify.atlassian.net/browse/KBV-294)

### Environment variables or secrets

- [ ] No environment variables or secrets were added or changed

- [ ] Documented in the [README](./blob/main/README.md)
- [ ] Added to deployment repository
- [ ] Added to local startup repository

### Other considerations

- [ ] Update [README](./blob/main/README.md) with any new instructions or tasks

### CODE REVIEW

File: src/app/address/controllers/addressConfirm.test.js
Comment (MattBidewell) [inline src/app/address/controllers/addressConfirm.test.js:90]: @merlinc A ref if you need to test any response in your unit tests.

My tests where failing because expect(res.redirect..) was being called before the test was.

**Comment by MattBidewell** [comment]
Reopened the PR for now but wish to talk about it before any merging.
Was comparing the differences in workflow with the passport and eager to get comments. @idrop @merlinc

This workflow is in line with UML diagram in API [here](https://github.com/alphagov/di-ipv-cri-address-api/blob/main/doc/api.puml)

**Comment by MattBidewell** [comment]
- Aligned with what passport is doing.

**Comment by MattBidewell** [comment]
I will tidy up commits once I get an approval on the code generically.

**Review by merlinc — DISMISSED** [review]
LGTM

**Review by merlinc — APPROVED** [review]

**Pull request merged**
